src/index.js

```diff
@@ -1,38 +1,16 @@
 import './style.css';
 
-const results = [
-  {
-    name: 'Ali',
-    score: '100',
-  },
-  {
-    name: 'Henry',
-    score: '100',
-  },
-  {
-    name: 'Elyor',
-    score: '100',
-  },
-  {
-    name: 'Zeenat',
-    score: '100',
-  },
-  {
-    name: 'Faizii',
-    score: '100',
-  },
-  {
-    name: 'Elmar',
-    score: '100',
-  },
-];
+let results = [];
 
 const table = document.querySelector('.table');
 
-function generateResult() {
+function generateResult(results) {
```

Comment:
**Changes Required**
- [ ] The use of `arrow function` is preferred to the `function` keyword. Let us refactor this into an ES6 arrow function. You can read more [here](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why you should prefer ES6 arrow functions to the `function keyword`.

Reply:
Hello @oxenprogrammer. Thank you for your comment. I refactor the generateResult function to be an arrow function as you can see in the image below.
![Capture103](https://user-images.githubusercontent.com/60292761/131811876-9eab2515-8ab3-4ff1-95eb-a11d1992c127.PNG)